### PR TITLE
project vertices on geodesic sphere primitive

### PR DESCRIPTION
**Pull request by ThibaultLejemble**
* **Please check if the PR fulfills these requirements**
- [x] Is the Pull-Request done against the right branch:
  - `master`: for hotfixes not impacting the API
  - `master-v1`: for changes related to the Radium Stable Release V1.
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

Be aware that the PR request cannot be accepted if it doesn't pass the Continuous Integration tests.

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
It fixes a regression introduced in 1c4fd16567313b56286db2d0c79f5bd6ebc2c2ea. 

* **What is the current behavior?** (You can also link to an open issue here)
The `MeshUtils::makeGeodesicSphere()` method does not project vertices on the sphere.

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: src/Core/Mesh/MeshPrimitives.cpp [hunk 1]
```diff
@@ -80,8 +89,22 @@ TriangleMesh makeSharpBox( const Aabb& aabb ) {
                          // Top face
                          aabb.corner( Aabb::TopLeftFloor ), aabb.corner( Aabb::TopLeftCeil ),
                          aabb.corner( Aabb::TopRightCeil ), aabb.corner( Aabb::TopRightFloor )
+    };
 
+    result.m_normals = {// Foor face
```
Comment (nmellado) [inline src/Core/Mesh/MeshPrimitives.cpp]: Typo: Floor

**Comment by ThibaultLejemble** [comment]
For now, I have only visually tested these changes by creating each primitive from a Radium plugin :
screenshot

**Comment by nmellado** [comment]
Maybe it's the right moment to clean this a bit..
Do we really need shapes with sharpe edges but smooth normals ?
@dlyr @MathiasPaulin

**Comment by nmellado** [comment]
I'm merging this PR as it fixes correctly the problem.
Further discussions about Radium's primitives can be posted in a dedicated issue.

**Pull request merged**